nemo/collections/asr/parts/submodules/rnnt_loop_labels_computer.py

```diff
@@ -630,7 +631,7 @@ def _partial_graphs_compile(self):
         with (
             torch.cuda.stream(stream_for_graph),
             torch.inference_mode(),
-            torch.cuda.graph(
+            checked_graph(
```

Comment:
Do we need `checked_graph` for partial graphs (without conditional nodes)?

Reply:
This is a very good point. No, we don't.

But I am considering doing your suggestion anyway for a fallback, in which case we wouldn't use checked_graph.